src/doc/src/reference/workspaces.md

```diff
@@ -30,6 +33,10 @@ In the `Cargo.toml`, the `[workspace]` table supports the following sections:
 * [`[replace]`](overriding-dependencies.md#the-replace-section) --- Override dependencies (deprecated).
 * [`[profile]`](profiles.md) --- Compiler settings and optimizations.
 
+## Creating a Workpaces
+
+todo!：Added example of creating a workspace.
+
```

Comment:
Due to the challenges in writing this documentation, I agree with @weihanglo about keeping each PR focused.  These changes will likely involve a lot of iteration and coupling them together in one PR will likely make this longer and more exhausting